notify/impl.go

```diff
@@ -885,8 +888,6 @@ func (n *Wechat) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 
 		u.RawQuery = parameters.Encode()
 
-		level.Debug(n.logger).Log("msg", "Sending Wechat message", "incident", key, "url", u.String())
```

Comment:
Removing this line since it could potentially leak `corpsecret` and `corpid` in the logs.